operate/client/src/modules/queries/userTasks/useUserTasksSearch.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useQuery} from '@tanstack/react-query';
+import {searchUserTasks} from 'modules/api/v2/userTasks/searchUserTasks';
+import {QueryUserTasksRequestBody} from '@vzeta/camunda-api-zod-schemas';
+
+const USER_TASKS_SEARCH_QUERY_KEY = 'userTasksSearch';
+
+function getQueryKey(params: QueryUserTasksRequestBody) {
+  return [USER_TASKS_SEARCH_QUERY_KEY, params];
+}
+
+const useUserTasksSearch = (params: QueryUserTasksRequestBody) => {
+  return useQuery({
+    queryKey: getQueryKey(params),
+    queryFn: () =>
+      searchUserTasks(params).then(({response, error}) => {
+        if (response !== null) return response;
```

Comment:
Nitpick, but to keep it more consistent with the codebase let's braces
```suggestion
        if (response !== null) {
          return response;
        }
```

Reply:
Updated